sdk/tvOS/OoyalaTVSkinSDK/Widgets/OOOoyalaTVButton.h

```diff
@@ -12,6 +12,6 @@
 
 @property (nonatomic) CGFloat OOFontSize;
 
-- (void)setupPlayPause: (BOOL)isPlay;
+- (void)changePlayingState: (BOOL)isPlay;
```

Comment:
can you change the variable to "isPlaying"?  isPlay is ambiguous - isPlayButtonShowing, or isVideoPlaying?